cmd/branch_protection_test.go

```diff
@@ -2,7 +2,7 @@ package cmd
 
 import (
 	"github-admin-tool/graphqlclient"
-	"io/ioutil"
+	"os"
```

Comment:
Why is this changing?

Reply:
Deprecation... https://pkg.go.dev/io/ioutil. so migrated to os mod which is a drop in replacement